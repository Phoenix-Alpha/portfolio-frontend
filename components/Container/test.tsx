import { shallow } from 'enzyme'
import toJSON from 'enzyme-to-json'

import { Container } from './index'

describe('<Container />', () => {
  it('should renders children', () => {
    const text = 'test'

    const wrapper = shallow(<Container>{text}</Container>)

    expect(wrapper.text()).toMatch(text)

    expect(toJSON(wrapper)).toMatchSnapshot()
  })
})
