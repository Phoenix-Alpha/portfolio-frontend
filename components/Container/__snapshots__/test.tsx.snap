// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Container /> should renders children 1`] = `
<div
  className="container px-3 max-w-screen-xl mx-auto"
>
  test
</div>
`;
